harper-core/dictionary.dict

```diff
@@ -51617,7 +51617,7 @@ worksheet/14MS
 workshop/~14MS
 workshy/5
 worksite/1S
-workspace/1
+workspace/1S
```

Comment:
I always include `M` with all nouns because it means `'s` and you can always have things like `my workspace's setup` etc. It could probably be done internally to every noun without an annotation but since `'s` is also used in contractions of `is` and `has` that might be the reason to make it explicit.